.github/workflows/schedule.yml

```diff
@@ -324,4 +324,4 @@ jobs:
           printf "[INFO] Notify ID: '%s'\n" "${{ env.NOTIFY_ID }}"
           printf "[INFO] Software Name: '%s'\n" "${{ env.SOFTWARE_NAME }}"
           printf "[INFO] Software Version: '%s'\n" "${{ env.SOFTWARE_VERSION }}"
-          ve1/bin/pytest tests/functional/step_defs/test_submitted_charts.py --log-cli-level=WARNING --tb=short
+          ve1/bin/pytest -n 4 tests/functional/step_defs/test_submitted_charts.py --log-cli-level=WARNING --tb=short
```

Comment:
where does 4 comes from why not 8 or 16 etc...

Reply:
```
parallel testing:
 - n=1: ~34m58s
 - n=2: ~17m45s
 - n=4: ~9m33s
 - n=8 failed (exceeded api call limit): {'message': 'API rate limit exceeded for user ID 83200018.', 'documentation_url': 'https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting'}
```
When n>=8 we are easier to hit API rate limit from Github (something we can optimize later to minimize API call footprint). Therefore it is reasonable to set n=4. See a test run here: https://github.com/zonggen/development/runs/4357928277?check_suite_focus=true

Reply:
again, great question. thanks for reviewing!